agent_api_rest/openapi.yaml

```diff
@@ -120,12 +102,12 @@ paths:
       summary: Standard OpenID Connect discovery endpoint for issuer metadata
       tags:
         - (proxied)
-  /auth/token:
+  /v1/auth/token:
```

Comment:
@daniel-mader I am not sure anymore whether we decided to go with the `/v1` part here as well as for `/v1/openid4vci/credential`.. I've added it since it is used in the `agent_api_rest` atm (and elsewhere) so with this change at least the `openapi.yaml` file will be consistent with the code.